src/api/Api.js

```diff
@@ -0,0 +1,31 @@
+import superagent from 'superagent';
+
+class Api {
```

Comment:
Any special reason why this is a class with static methods? Could as well be only functions?